src/Utils.elm

```diff
@@ -0,0 +1,64 @@
+module Utils exposing (timeAgo)
+
+import Time exposing (Time)
+
+
+timeAgo : Time -> Time -> String
+timeAgo time now =
+    let
+        seconds =
+            round ((now - time) / 1000)
+
+        minutes =
+            seconds // 60
+
+        hours =
+            minutes // 60
+
+        days =
+            hours // 24
+
+        weeks =
+            days // 7
+
+        months =
+            days // 30
```

Comment:
You use `30` for a month here, but you test with `31`. Should the test be on `30` too?